demo/compose.yaml

```diff
@@ -1,32 +1,33 @@
-# Quick Start
-# docker compose up
+x-common-config:
+  &common-config
+  network_mode: host
+  ipc: host
+  env_file: .env
 
 services:
-
   rplidar:
-    image: husarion/rplidar:humble
-    restart: unless-stopped
-    network_mode: host
-    ipc: host
+    # image: husarion/rplidar:humble
+    build:
+      context: ..
+      dockerfile: Dockerfile
+    <<: *common-config
     devices:
       - /dev/ttyUSB0
-    environment:
-      - ROS_DOMAIN_ID=${ROS_DOMAIN_ID:-0}
-      - RMW_IMPLEMENTATION=${RMW_IMPLEMENTATION:-rmw_fastrtps_cpp}
     command: >
       ros2 launch sllidar_ros2 sllidar_launch.py
```

Comment:
This is the original launch. Passing the arguments gives no effect. Shouldn't be here `husarion_utils/rplidar.launch.py`?